### PR TITLE
Ci testing

### PR DESCRIPTION
**Pull request by linusaaberg**
Changed SmartCar.ino, initialize the inputhandler before loop. Relates to #4  and #3.
Also set up a CI for the repo and added build icon to readme.
Added vscode directories to .gitignore.
Branch also includes a PlatformIO.ini file which changes scr-dir to SmartCar directory.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+    language: c
+    before_install:
+      - source <(curl -SLs https://raw.githubusercontent.com/adafruit/travis-ci-arduino/master/install.sh)
+    install:
+      - arduino --install-library "ESP32 AnalogWrite"
+      - arduino --install-library "ServoESP32"
+      - arduino --install-library "Smartcar shield"
```
Comment (platisd) [inline .travis.yml:7]: I believe just this should be sufficient

Comment (linusaaberg) [reply .travis.yml:7]: How do you mean?  To remove everything else? Don´t i need to install the travic-ci-arduino? And the script?

Comment (platisd) [reply .travis.yml:7]: No, I mean that you can most likely remove the following two lines:
```yaml
      - arduino --install-library "ESP32 AnalogWrite"
      - arduino --install-library "ServoESP32"
```

Comment (linusaaberg) [reply .travis.yml:7]: So I tried it out and the builds fails:
`/home/travis/Arduino/libraries/Smartcar_shield/src/runtime/arduino_runtime/ArduinoRuntime.cpp:7:25: fatal error: analogWrite.h: No such file or directory

compilation terminated.

exit status 1`

Comment (platisd) [reply .travis.yml:7]: Oh, that's sad/strange. I thought this command would also install the dependencies too and is equivalent to using the library manager on the IDE. Oh well, never mind then.

**Review by Brother-Blue — APPROVED** [review]

**Comment by platisd** [comment]
On an unrelated note, I would suggest you change your local git username to the one you have on GitHub (i.e. from `Linusab` to `LinusAaberg`) so your commits can be tracked easily.

**Pull request merged**